graduates.html

```diff
@@ -41,6 +41,237 @@ <h1 class="header-h1 grid-item">Refcode</h1>
+        </ul>
+      </div>
+      <div class="student">
+        <h4 class="student-name">Bezawit</h4>
+        <div class="student-photo">
+          <img src="media/Bezawit.png"/>
+        </div>
+        <ul class="student-links">
+          <li>Website: <a href="http://bezu.ga/" rel="external">bezu.ga</a></li>
+          <li><a href="https://github.com/bez" rel="external">GitHub Profile</a></li>
+          <li><a href="#" rel="external">LinkedIn Profile</a></li>
+        </ul>
+      </div>
+
+      <div class="student">
+        <h4 class="student-name">Bizuayehu</h4>
+        <div class="student-photo">
+          <img src="media/Bizuayehu.png"/>
+        </div>
+        <ul class="student-links">
+          <li>Website: <a href="http://bizuayehu.ga/" rel="external">bizuayehu.ga</a></li>
+          <li><a href="https://github.com/bizuayehu1968/Soli" rel="external">GitHub Profile</a></li>
+          <li><a href="#" rel="external">LinkedIn Profile</a></li>
+        </ul>
+      </div>
+
+      <div class="student">
+        <h4 class="student-name">James</h4>
+        <div class="student-photo">
+          <img src="media/James.png"/>
+        </div>
+        <ul class="student-links">
+          <li>Website: <a href="http://jkalunga.com/" rel="external">jkalunga.com</a></li>
+          <li><a href="https://github.com/jskalu" rel="external">GitHub Profile</a></li>
+          <li><a href="https://www.linkedin.com/in/james-kalunga-a9786368" rel="external">LinkedIn Profile</a></li>
+        </ul>
+      </div>
+
+      <div class="student">
+        <h4 class="student-name">Liya</h4>
+        <div class="student-photo">
+          <img src="media/Liya.png"/>
+        </div>
+        <ul class="student-links">
+          <li>Website: <a href="http://liya2019.ga/" rel="external">liya2019.ga</a></li>
+          <li><a href="https://github.com/Liya2019" rel="external">GitHub Profile</a></li>
+          <li><a href="#" rel="external">LinkedIn Profile</a></li>
+        </ul>
+      </div>
+
+      <div class="student">
+        <h4 class="student-name">Nezira</h4>
+        <div class="student-photo">
+          <img src="media/Nezira.png"/>
+        </div>
+        <ul class="student-links">
+          <li>Website: <a href="http://nezirausmael.ga/" rel="external">nezirausmael.ga</a></li>
+          <li><a href="https://github.com/UmmuAyyub" rel="external">GitHub Profile</a></li>
+          <li><a href="https://www.linkedin.com/in/nezira-usmael-6358b3182" rel="external">LinkedIn Profile</a></li>
+        </ul>
+      </div>
+
+      <div class="student">
+        <h4 class="student-name">Nisar</h4>
+        <div class="student-photo">
+          <img src="media/Nisar.png"/>
+        </div>
+        <ul class="student-links">
+          <li>Website: <a href="http://nisar.ga/" rel="external">nisar.ga</a></li>
+          <li><a href="https://github.com/nisarfriend/NEW-Repository" rel="external">GitHub Profile</a></li>
```

Comment:
This link should go to the profile, not the repo. So just:

https://github.com/nisarfriend/